core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/controller/cart/BroadleafCartController.java

```diff
@@ -302,11 +299,7 @@ public String updateQuantity(HttpServletRequest request, HttpServletResponse res
         
         if (isAjaxRequest(request)) {
             Map<String, Object> extraData = new HashMap<>();
-            if(useSku) {
-                extraData.put("skuId", itemRequest.getSkuId());
-            } else {
-                extraData.put("productId", itemRequest.getProductId());
-            }
+            extraData.put("productId", itemRequest.getProductId());
```

Comment:
This is not quite right, you should just always set the `skuId` based on the `itemRequest` anyway.